addons/block_code/blocks/input/characterbody2d_move.tres

```diff
@@ -10,7 +10,7 @@ description = "Move the character up, down, left, and right with the keyboard us
 category = "Input"
 type = 2
 variant_type = 0
-display_template = "Move with keys {up: STRING} {down: STRING} {left: STRING} {right: STRING} with speed {speed: VECTOR2}"
+display_template = "move with keys {up: STRING} {down: STRING} {left: STRING} {right: STRING} with speed {speed: VECTOR2}"
```

Comment:
```suggestion
display_template = "move with keys {up: STRING} {down: STRING} {left: STRING} {right: STRING} at speed {speed: VECTOR2}"
```

Reply:
I have to say that I'm not a fan of this block. I would prefer one block that can configure the player 1/2 keys, to work with the existing "move with player 1/2 buttons ...". But a discussion for another day.